understanding/21/reflow.html

```diff
@@ -12,7 +12,7 @@ <h1>Understanding Reflow</h1>
         <h2>Intent of this Success Criterion</h2>
         <p>The intent of this Success Criterion is to support people with low vision who need to enlarge text and read it in a single column.  When the browser zoom is used to scale content to 400%, it reflows - i.e., it is presented in one column so that scrolling in more than one direction is not necessary.</p>
       <p>For people with low vision, enlarged text with reflow enables reading. It is critical. Enlargement enables perception of characters. Reflow enables tracking. Tracking is following along lines of text, including getting from the end of one line to the beginning of the next line.</p>
-      <p>Avoiding the need to scroll in the direction of reading in order to reveal lines that are cut off by the viewport is important, because such scrolling significantly increases the effort required to read. It is also important that content is not hidden off-screen. For example, zooming into a vertically scrolling page should not cause content to be hidden to one side.</p>
+      <p>Avoiding the need to scroll in the direction of reading in order to reveal lines that overflow the viewport is important, because such scrolling significantly increases the effort required to read. It is also important that content is not hidden off-screen and completely cut off. For example, using the browser's zoom functionality on a vertically scrolling page should not cause content to be hidden to one side, with no way of reaching it.</p>
```

Comment:
```suggestion
      <p>Avoiding the need to scroll in the direction of reading in order to reveal lines that overflow the viewport is important, because such scrolling significantly increases the effort required to read. It is also important that content is not hidden off-screen and truncated. For example, using the browser's zoom functionality on a vertically scrolling page should not cause content to be hidden to one side, with no way of reaching it.</p>
```